model.js

```diff
@@ -44,10 +44,30 @@ function getArticalsbyId(id) {
       return result.rows[0];
     });
 }
+
+//CORE: PATCH /api/articles/:article_id Model
+const updateArticleVotesModel = (article_id,inc_votes)=>{
+  return db
+  .query(
+    `UPDATE articles 
+     SET votes = votes + $1 
+     WHERE article_id = $2 
+     RETURNING *;`,
+    [inc_votes, article_id]
+  )
+  .then((result) => {
+    if (result.rows.length === 0) {
+      return null; 
```

Comment:
You should return a promise.reject here with an error status and message. You can then send that to next and in your error handler send a response to the client